bot.js

```diff
@@ -62,10 +62,99 @@ const sendUpdateMessage = (channel, text, name, args) => {
     });
 };
 
-const parseCommand = (text) => {
-    const args = text.toLowerCase().split(' ').filter((arg) => {
+const extractArgs = (text) => {
```

Comment:
Do you think you could add a jsdoc showing one or two examples of inputs/outputs? Makes it more clear since the implementation is pretty low-level

Reply:
How's that?